src/sections/Statistics/style.module.scss

```diff
@@ -0,0 +1,27 @@
+@use '../../styles/vars.scss' as vars;
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  margin-top: 10px;
+  margin-bottom: 10px;
+
+  .innerContainer {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    flex: 1;
+  }
+  
+
+  .diamond {
+    position: relative;
+    aspect-ratio: 1;
```

Comment:
Maybe edit the aspect ratio to match the figma so the diamond guy is the right size